src/Common/RJSF/templates/ArrayFieldTemplate.tsx

```diff
@@ -19,14 +19,20 @@ import { getTemplate, getUiOptions, ArrayFieldTemplateProps, ArrayFieldTemplateI
 import { FieldRowWithLabel } from '../common/FieldRow'
 import { DO_NOT_SHOW_LABEL } from '../constants'
 
-const ActionButton = ({ canAdd, onAddClick, disabled, readonly, uiSchema, registry }) => {
+const ActionButton = ({ label, canAdd, onAddClick, disabled, readonly, uiSchema, registry }) => {
     const {
         ButtonTemplates: { AddButton },
     } = registry.templates
 
     return (
         canAdd && (
-            <AddButton onClick={onAddClick} disabled={disabled || readonly} uiSchema={uiSchema} registry={registry} />
+            <AddButton
+                label={label}
```

Comment:
What's the reasoning for adding this?

Reply:
The Add button will now show Add paths if 'paths' is the label/title/key of the object or array